### PR TITLE
runcon: return UResult from uumain() function

### PR DESCRIPTION
**Pull request by jfinkels**
See #2464.

I don't know how to test this one, so please review with care.

### CODE REVIEW

File: src/uu/runcon/src/runcon.rs [hunk 1]
```diff
@@ -65,39 +57,29 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
                 match r.kind {
                     clap::ErrorKind::HelpDisplayed | clap::ErrorKind::VersionDisplayed => {
                         println!("{}", r);
-                        return error_exit_status::SUCCESS;
+                        set_exit_code(error_exit_status::SUCCESS);
```
Comment (tertsdiepraam) [inline src/uu/runcon/src/runcon.rs:60]: 0 is automatically set as the exit code when `Ok(())` is returned

File: src/uu/runcon/src/runcon.rs [hunk 2]
```diff
@@ -106,34 +88,30 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
...
-
-                show_error!("{}", report_full_error(&err));
-                return exit_status;
-            } else if let Err(r) = print_current_context() {
-                show_error!("{}", report_full_error(&r));
-                return error_exit_status::ANOTHER_ERROR;
+                }
+                None => {
+                    print_current_context().map_err(RunconError::new)?;
+                    set_exit_code(error_exit_status::SUCCESS);
```
Comment (tertsdiepraam) [inline src/uu/runcon/src/runcon.rs:109]: Same here

**Review by tertsdiepraam — APPROVED** [review]
I also can't really test this, but it looks good to me

File: src/uu/runcon/src/runcon.rs (hunk 1 above)
Comment (jfinkels) [reply src/uu/runcon/src/runcon.rs:60]: I suppose it is safe to assume that `error_exit_status::SUCCESS` will always be 0 :). I'll make that change.

https://docs.rs/libc/latest/libc/constant.EXIT_SUCCESS.html

**Comment by jfinkels** [comment]
I have made the suggested changes and rebased.

**Pull request merged**